amarillo/services/secrets.py

```diff
@@ -1,9 +1,8 @@
-from typing import Dict
-from pydantic import Field
+from pydantic import Field, ConfigDict
 from pydantic_settings import BaseSettings
-
 # Example: secrets = { "mfdz": "some secret" }
 class Secrets(BaseSettings):
+    model_config = ConfigDict(extra='allow')
```

Comment:
Could you leave a comment, why you inserted extra='allow'? (I suppose this allows extension by plugins? In this case: What should a plugin provider do to extend?)

Reply:
Done